Src/WitsmlExplorer.Frontend/components/Sidebar/WellboreItem.tsx

```diff
@@ -118,6 +118,7 @@ const WellboreItem = (props: WellboreItemProps): React.ReactElement => {
       labelText={wellbore.name}
       onLabelClick={onLabelClick}
       onIconClick={onIconClick}
+      isActive={wellbore.isActive === "true"}
```

Comment:
Will this catch all string values from Store? e.g. "0" an "1" which is not prohibited as an xml boolean ref earlier note: 

> For Boolean items, the XML Schema standard supports use of both “true” and “1” and “false” and “0”. WITSML is consistent with XML, so also supports use of both.

Reply:
This will not catch all string values. I need to either compare 0 and 1 as well or turn it into a boolean somewhere.